make-it/EmMakefile

```diff
@@ -10,6 +10,7 @@ SITEDIR=$(VIREO_SDK_DIR)/Documents/gh-pages/playground
 TARGETDIR=$(VIREO_SDK_DIR)/dist
 CORESOURCEDIR=$(VIREO_SDK_DIR)/source/core
 IOSOURCEDIR=$(VIREO_SDK_DIR)/source/io
+UTSOURCEDIR=$(VIREO_SDK_DIR)/source/unittest
```

Comment:
I guess at some point in the future we want a release build where this is not compiled in?